console/billing/how-to/change-payment-method.mdx

```diff
@@ -0,0 +1,35 @@
+---
+meta:
+  title: How to change your default payment method
+  description: This page explains how to change your default payment method
+content:
+  h1: How to change your payment method
+  paragraph: This page explains how to change your default payment method
+tags: billing change default payment 
+dates:
+  validation: 2024-02-15
+  posted: 2024-02-15
+categories:
+  - billing
+---
+
+If you have registered both a **credit card** and a **SEPA mandate**, you can choose the default payment method you wish to use for your Scaleway account.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com).
+</Message>
+
+1. Click **Billing** in the Organization drop-down menu to go to the **Billing Overview** page.
+    <Message type="tip">
+      Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking **Go to Billing** above the **Current consumption** area.
```

Comment:
Alternatively, you can reach the [billing page](https://console.scaleway.com/organization/billing/overview) from your [Organization Dashboard](https://console.scaleway.com/organization) by clicking Go to Billing above the Current consumption area.